docker/mapfish-print-v2/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM terrestris/tomcat:8.5.37
+
+RUN apk -U upgrade --update && \
+    apk add curl && \
+    rm -rf $CATALINA_HOME/webapps/*
+
+RUN curl -jkSL -o /tmp/print.war http://central.maven.org/maven2/org/mapfish/print/print-servlet/2.1.4/print-servlet-2.1.4.war && \
```

Comment:
Please switch to `https` (e.g. by using `https://repo.maven.apache.org/maven2/org/mapfish/print/print-servlet/`) and make version configurable.

Reply:
Changed and configured in 1e751c0